docs/src/getting-started/installation.md

```diff
@@ -53,12 +45,6 @@ On Linux systems you may need to install additional dependencies if `cargo insta
 sudo apt-get update && sudo apt-get upgrade && sudo apt-get install -y pkg-config build-essential libudev-dev
 ```
 
-To install the JavaScript package.
-
-```bash
-npm install -g @project-serum/anchor
-```
-
 Make sure your `NODE_PATH` is set properly so that globally installed modules
 can be resolved.
```

Comment:
Since we will not have globally installed packages, can we remove this line?

Reply:
Missed that one, thanks!